espnet/asr/pytorch_backend/asr_mulenc.py

```diff
@@ -0,0 +1,536 @@
+#!/usr/bin/env python3
+
+"""Define asr module for multi-encoder network."""
```

Comment:
Is it possible to unify `asr_multienc.py` features into `asr.py`? I think diff is not so large.

Reply:
I could try to unify both togther!